app/views/application-process.html

```diff
@@ -0,0 +1,94 @@
+
+      <p class="govuk-body">For courses starting in September 2023, you can start applying a year before they start in October 2022.</p>
+
+      <p class="govuk-body">Most teacher training courses are one year and start in September when the academic year in England starts. Part time courses are usually 2 years.</p>
+      <br>
+        <table class="govuk-table">
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              <th scope="col" class="govuk-table__header">Date and time</th>
+              <th scope="col" class="govuk-table__header">What happens</th>
+            </tr>
+          </thead>
+          <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
+             <td class="govuk-table__cell">4&nbsp;October&nbsp;2022 at 9am</td>
+              <td class="govuk-table__cell">Start <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses" class="govuk-link">finding postgraduate teacher training courses</a> you might want to apply to.</td>
+            </tr>
+            <tr class="govuk-table__row">
+             <td class="govuk-table__cell">11&nbsp;October&nbsp;2022 at 9am</td>
+              <td class="govuk-table__cell">Start applying for courses. You can apply to 4 courses at a time. Courses can fill up quickly, so you should apply as soon as you can.<br><br>You should make sure each course you choose is offered by a different training provider so you can have more chance of success.<br><br>Training providers will have 40 working days to decide to offer you a place on a course. Holiday periods in England over December and March or April are not included in the 40 working days. Each course you apply to will show the date for when providers need to respond by.<br><br>If training providers do not respond in time, your application will become unsuccessful.<br><br>You can apply to 4 courses again if all your applications become unsuccessful.<br><br>You’ll have 10 working days to respond to any offers you receive. If you do not respond, the offer will be automatically declined.</td>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell">1 July 2023</td>
+              <td class="govuk-table__cell">The time training providers have to make a decision about your application is reduced from 40 working days to 20 working days.</td>
+            </tr>
+              <tr class="govuk-table__row">
+              <td class="govuk-table__cell">5 September 2023 at 6pm</td>
+              <td class="govuk-table__cell">If you have not already applied for teacher training, you will not be able to apply after 5 September 2023.<br><br> If you've already applied, but your applications were unsuccessful, you can continue to apply for courses until 19 September 2023 at 6pm.</td>
+            </tr>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell">27 September 2023 at 11:59pm</td>
```

Comment:
If we add `&nbsp;` (non-breaking space) here, then the date won't wrap across multiple lines (forcing the column to be slightly wider), which might be more readable?

```suggestion
              <td class="govuk-table__cell">27&nbsp;September&nbsp;2023 at 11:59pm</td>
```

Reply:
Ah nice! I did not know this!